lib/index.js

```diff
@@ -65,7 +65,15 @@ Strategy.prototype.authenticate = function (req, options) {
 };
 
 Strategy.prototype.authorizationParams = function(options) {
```

Comment:
We need to validate if we have an options object before trying to access it, e.g.

```
options = options || {};
```